media-video/gnash/gnash-0.8.11~dev.recipe

```diff
@@ -0,0 +1,136 @@
+CHECKSUM_SHA256="8437e3bae8163ed02a66028203408eb8a54bd4a84e256a43f0c26b87e8c882e6"
+SOURCE_FILENAME="gnash-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="gnash-$srcGitRev"
+PATCHES="gnash-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 !x86 !x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 ?x86"
+
+GLOBAL_WRITABLE_FILES="
+	settings/gnashpluginrc auto-merge
+	settings/gnashrc auto-merge
+	"
+
+PROVIDES="
+	gnash$secondaryArchSuffix = $portVersion
+	cmd:gnash$secondaryArchSuffix
+	cmd:gprocessor$secondaryArchSuffix
+	cmd:rtmpget$secondaryArchSuffix
+	cmd:sdl_gnash$secondaryArchSuffix
+	#app:Gnash = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libagg$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix
+	lib:libboost_date_time$secondaryArchSuffix
+	lib:libboost_iostreams$secondaryArchSuffix
+	lib:libboost_program_options$secondaryArchSuffix
+	lib:libboost_serialization$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	lib:libboost_thread$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgif$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libsdl$secondaryArchSuffix
+	lib:libsdl_sound$secondaryArchSuffix
+	lib:libspeex$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	boost${secondaryArchSuffix}_devel
+	devel:libagg$secondaryArchSuffix
+	devel:libavcodec$secondaryArchSuffix
+	devel:libboost_date_time$secondaryArchSuffix
+	devel:libboost_iostreams$secondaryArchSuffix
+	devel:libboost_program_options$secondaryArchSuffix
+	devel:libboost_serialization$secondaryArchSuffix
+	devel:libboost_system$secondaryArchSuffix
+	devel:libboost_thread$secondaryArchSuffix
+	devel:libbz2$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libexpat$secondaryArchSuffix
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgif$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libltdl$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libsdl$secondaryArchSuffix
+	devel:libsdl_sound$secondaryArchSuffix
+	devel:libspeex$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:automake
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sdl_config$secondaryArchSuffix
```

Comment:
Not sure this is needed so rebuilding without it (should be already provided by libSDL)